src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationServiceException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_LENGTH = 6;
+    private static final int MIN_AGE = 18;
+    private static final String VALID_LOGIN = "123456";
+    private static final String VALID_PASSWORD = "654321";
+    private static final String NOT_VALID_LOGIN = "JAVA";
+    private static final String NOT_VALID_PASSWORD = "1234";
+    private static final int NOT_VALID_AGE = 13;
+
+    private static RegistrationService registrationService;
+    private User user = new User(VALID_LOGIN, VALID_PASSWORD, MIN_AGE);
```

Comment:
it is initialized in the `beforeEach` anyway
```suggestion
    private User user;
```

Reply:
fixed